incubator/kafka/README.md

```diff
@@ -0,0 +1,111 @@
+# Apache Kafka Helm Chart
+
+This is an implementation of Kafka PetSet found here:
+
+ * https://github.com/Yolean/kubernetes-kafka
+
+## Pre Requisites:
+
+* Kubernetes 1.3 with alpha APIs enable
```

Comment:
You are also using storage classes as well.  You probably should mention both of those.